src/utils/lineGraph.go

```diff
@@ -66,7 +66,11 @@ func (l *LineGraph) Draw(buf *ui.Buffer) {
 	// sort the series so that overlapping data will overlap the same way each time
 	seriesList := make([]string, len(l.Data))
 	i := 0
-	l.MaxVal = 1
+
+	if l.MaxVal == 0 {
+		l.MaxVal = 1
+	}
```

Comment:
This seems a little redundant right? Why not just assign default as 1?

Reply:
Oh I explained this somewhere above. MaxVal can become zero if the data it's trying to graph has the least value 0.

Reply:
Causes segfaults